doc/source/rse_expressions.rst

```diff
@@ -38,7 +37,7 @@ Rucio allows to test RSE Expressions, using the command **list-rses**. The most
 
 4) Or all the Spanish sites::
 
-    jbogadog@lxplus0058:~$ rucio list-rses --expression SPAINSITES
+    jbogadog@lxplus0058:~$ rucio list-rses --expression ESSITES
```

Comment:
Change to non-ATLAS examples

Reply:
Do you really think it is necessary for these? Already looks so generic to me.

Reply:
i thought SPAINSITES was nice, ESSITES looks a bit weird to me